docs/manuals/source/TasksAndConcepts/DataEncryption.rst

```diff
@@ -69,35 +69,34 @@ The various algorithms are exposed via an entirely re-usable, OpenSSL-agnostic A
 Generating Private/Public Encryption Keys
 -----------------------------------------
 
-:index:`\ <single: Encryption; Generating Private/Public Encryption Keypairs>`\ 
+:index:`\ <single: Encryption; Generating Private/Public Encryption Keypairs>`\
 
-Generate a Master Key Pair with:
+Generate a public/private key-pair with:
 
 
 
 ::
 
-     openssl genrsa -out master.key 2048
-     openssl req -new -key master.key -x509 -out master.cert
+     openssl genrsa -out master.priv.key 2048
+     openssl req -new -key master.priv.key -x509 -out master.publ.key
```

Comment:
I would call it "pub", not "publ", as this is usually the name in ssh like id_dsa.pub